src/components/ChattyLLM/ChattyLLMInputForm.vue

```diff
@@ -0,0 +1,674 @@
+<template>
+	<div class="container">
+		<NcAppNavigation>
+			<NcAppNavigationList>
+				<NcAppNavigationNew :text="t('assistant', 'New conversation')"
+					type="secondary"
+					@click="newSession">
+					<template #icon>
+						<PlusIcon :size="20" />
+					</template>
+				</NcAppNavigationNew>
+				<div v-if="sessions == null" class="unloaded-sessions">
+					<NcLoadingIcon :size="30" />
+					{{ t('assistant', 'Loading conversations...') }}
+				</div>
+				<div v-else-if="sessions != null && sessions.length === 0" class="unloaded-sessions">
+					{{ t('assistant', 'No conversations yet') }}
+				</div>
+				<NcAppNavigationItem
```

Comment:
It just happens when there is another NcAppNavigation in the page.

Reply:
nope, pulled server master just now. Works on both chromium and firefox.

Reply:
> It just happens when there is another NcAppNavigation in the page.

yes it does, let me see if z-index or something can do the trick

Reply:
hmm, I don't even have the popover mounted in my DOM. Can't really point at what's going wrong.

Reply:
While using the `NcAppNavigation` outside of the `NcContent` and within a dialog might work it is not supported usage of the component.

But a real problem here: It **must not** be used more than once on a page, it will break and even create invalid HTML content of the page. This will also cause any accessibility tests on that page to fail.

Instead the expected solution would probably be either:
```html
<NcDialog ...>
  <template #navigation>
    <ul>
      <NcListItem ...>
      <!-- .... -->
    </ul>
  </template>
  <!-- Content -->
</NcDialog>
```

or

```html
<NcDialog ...>
  <your-content-wrapper>
    <ul class="your-navigation">
      <NcListItem ...>
      <!-- .... -->
    </ul>
    <!-- Other content -->
  </your-content-wrapper>
</NcDialog>
```